includes/admin/meta-boxes/views/html-variation-admin.php

```diff
@@ -195,6 +195,24 @@
 						)
 					);
 
+					woocommerce_wp_text_input(
```

Comment:
I'd suggest in this version we only add this at product level, not variation level. 3 levels of inheritance (global -> product -> variation) could get confusing. If it's requested after launch we can revisit.